app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidator.java

```diff
@@ -1,23 +1,32 @@
 package dev.andrylat.raqimbek.bankingutils;
 
+import lombok.NonNull;
+
 import java.util.Arrays;
 import java.util.List;
 import java.util.ArrayList;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
-public class CardValidator {
-  public CardValidationInfo checkCardNumber(String input) {
-    var errors = generateErrors(input);
+public class CardValidator implements Validator {
+  @NonNull
+  public CardValidationInfo validate(@NonNull List<String> inputList) {
+    var errors = new ArrayList<String>();
+
+    inputList.forEach(
+        input -> {
+          var _errors = generateErrors(input);
```

Comment:
what is the hell with naming? Why is it started with "_"? What is naming notation followed?